packages/web/src/components/composer-input/ComposerInput.tsx

```diff
@@ -290,7 +290,7 @@ export const ComposerInput = (props: ComposerInputProps) => {
       const cursorPosition = textarea.selectionStart
 
       if (isUserAutocompleteActive) {
-        if (e.key === ENTER_KEY || e.key === TAB_KEY) {
+        if (e.key === TAB_KEY) {
```

Comment:
ENTER_KEY removed here because it's already handled by the OptionKeyHandler right?

Reply:
yeah exactly